src/main/scala/mesosphere/marathon/api/v2/conversion/ContainerConversion.scala

```diff
@@ -0,0 +1,42 @@
+package mesosphere.marathon.api.v2.conversion
```

Comment:
I'm kind of thinking this should be in mesosphere/marathon/raml (or mesosphere/marathon/core/raml)